lib/cli.js

```diff
@@ -16,6 +16,7 @@ const yargv = yargs
   .nargs('features', 1)
   .alias('features', 'f')
   .describe('transformer', 'path to module which exports a code transformer function')
+  .describe('preprocessor', 'path to module which exports a map function that operates on each Test262Test object before it executed')
```

Comment:
Note that this could be used as a more powerful mechanism than just `transformer`, since the preprocessor could do things like: 

```js
const babel = require("@babel/core");
const options = {
  cwd: __dirname,
  presets: [
    [
      "@babel/preset-env",
      {
        spec: true
      }
    ]
  ]
};

module.exports = function(test) {
  try {
    test.contents = babel.transform(test.contents, options).code;
  } catch (error) {
    test.preprocessor = {
      autofail: {
        result: {
          stderr: error.toString(),
          stdout: '',
          error: null
        }
      }
    };
  }

  return test;
};
```